draft-barnes-acme.md

```diff
@@ -133,9 +133,6 @@ Subject Public Key:
 Account Key Pair:
 : A key pair for which the ACME server considers the holder of the private key authorized to manage certificates for a given identifier.  The same key pair may be authorized for multiple identifiers.
 
-Recovery Token:
```

Comment:
We should not remove the Recovery Token; let's have the client send an HMAC of one, which the server can record.